packages/devtools_app/test/standalone_ui/ide_shared/property_editor_test.dart

```diff
@@ -2,7 +2,7 @@
 // Use of this source code is governed by a BSD-style license that can be
 // found in the LICENSE file.
 
-import 'package:devtools_app/devtools_app.dart';
+import 'package:devtools_app/src/service/service_manager.dart';
```

Comment:
fine to leave as is but just FYI, we can use the `devtools_app.dart` import in tests, but prefer not to in src/ code

Reply:
Good to know! Switched to devtools_app import.